specification/storagecache/resource-manager/readme.md

```diff
@@ -101,8 +112,6 @@ See configuration in [readme.typescript.md](./readme.typescript.md)
 
 See configuration in [readme.ruby.md](./readme.ruby.md)
 
-
 ## AzureResourceSchema
 
 See configuration in [readme.azureresourceschema.md](./readme.azureresourceschema.md)
```

Comment:
Need to be updated to reflect the 2020-10-01 version, you can check the existing blocks in the readme.

Reply:
Updated the azure resource schema file in next commit.

Reply:
Fixed in next commit.